codec/decoder/targets.mk

```diff
@@ -76,6 +76,24 @@ endif
 OBJS += $(DECODER_OBJSMIPS_MMI)
 OBJS += $(DECODER_OBJSMIPS_MSA)
 
+DECODER_ASM_LOONGARCH_LSX_SRCS=\
+
+DECODER_OBJSLOONGARCH_LSX += $(DECODER_ASM_LOONGARCH_LSX_SRCS:.c=.$(OBJ))
+
+DECODER_ASM_LOONGARCH_LASX_SRCS=\
+
+DECODER_OBJSLOONGARCH_LASX += $(DECODER_ASM_LOONGARCH_LASX_SRCS:.c=.$(OBJ))
+ifeq ($(ASM_ARCH), loongarch)
+ifeq ($(ENABLE_LSX), Yes)
+DECODER_OBJS += $(DECODER_OBJSLOONGARCH_LSX)
+endif
+ifeq ($(ENABLE_LASX), Yes)
+DECODER_OBJS += $(DECODER_OBJSLOONGARCH_LASX)
+endif
+endif
+OBJS += $(DECODER_OBJSLOONGARCH_LSX)
```

Comment:
should this line be removed?
I think DECODER_OBJSLOONGARCH_LSX add to DECODER_OBJS is ok. no need to add it to OBJS

Reply:
Yes, you are right. Will update.